pdk/1.0/modules/ROOT/pages/policies-pdk-architecture.adoc

```diff
@@ -4,15 +4,15 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-The Policy Development Kit (PDK) consists of the:
+Flex Gateway Policy Development Kit (PDK) consists of the:
 
 * <<anypoint-cli-pdk-plugin>>: Generates and distributes the policy to Exchange
 * <<policy-template>>: Scaffolds new custom policies
 * <<sdk-building-tools, SDK Building Tools>>: Simplifies the development process
 
 == Anypoint CLI PDK Plugin
 
-The Anypoint CLI PDK plugin creates the PDK project and uploads policy assets to Exchange. After creating the project, PDK provides a simplied set of commands in a `Makefile` to complete all following development steps.
+The Anypoint CLI PDK plugin creates the PDK project and uploads policy assets to Exchange. After creating the project, PDK provides a simplified set of commands in a `Makefile` to complete all following development steps.
```

Comment:
Uploading to Exchange only applies to Connected Mode. Does this need to be clarified here?
```suggestion
The Anypoint CLI PDK plugin creates the PDK project and uploads policy assets to Exchange (for Connected Mode only). After creating the project, PDK provides a simplified set of commands in a `Makefile` to complete all following development steps.
```